flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDeserializationSchema.java

```diff
@@ -339,7 +365,7 @@ private Time convertToTime(Object object) {
         return new Time(millis - LOCAL_TZ.getOffset(millis));
     }
 
-    private Timestamp convertToTimestamp(Object object, boolean isMicros) {
+    private Timestamp convertToTimestamp(Object object, boolean isMicros, boolean legacyMapping) {
```

Comment:
could you change all `legacyMapping ` in this PR, we’d better use `legacyTimestampMapping `